site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -0,0 +1,174 @@
+
+{{< single-title-imgs
+    "Dark and glitchy (Super Mario Odyssey)"
+    "./smo-bug.png"
+    "./smo-fix.png"
+  >}}
+
+We are working on an [universal implementation on Vulkan](https://github.com/yuzu-emu/yuzu/pull/4150) that requires  the `VK_EXT_extended_dynamic_state` extension, which is not available on all GPU vendors' drivers yet. At the time of this writing, only the Intel Mesa `ANV` Linux driver and the latest Nvidia Vulkan Beta driver support it.
+
+A rare bug was affecting `Xenoblade Chronicles 2`, causing fantastic vertex explosions on time of the day transitions. [Rodrigo](https://github.com/ReinUsesLisp) managed to find the cause and by [improving the shader cache](https://github.com/yuzu-emu/yuzu/pull/4194) a bit, managed to solve the issue.
+
+{{< single-title-imgs
+    "Exactly the same spot (Xenoblade Chronicles 2)"
+    "./xc2-bug.png"
+    "./xc2-fix.png"
+  >}}
+  
+The Shark always gets a mention. By [implementing RendererInfo](https://github.com/yuzu-emu/yuzu/pull/4080) in yuzu's audio emulation, [ogniK](https://github.com/ogniK5377) solved the softlocks affecting `Animal Crossing: New Horizons` in the credits sequence and `The Legend of Zelda: Link's Awakening` in several areas of the game. Now you can enjoy both games without the fear of getting stuck, and hear some extra sound effects too!
+
+{{< single-title-imgs
+    "Nothing to report! (Animal Crossing: New Horizons & The Legend of Zelda: Link's Awakening)"
+    "./acnh-credits.png"
+    "./zla-mermaid.png"
+  >}}
+
+Additionally, also thanks to [ogniK](https://github.com/ogniK5377), `Animal Crossing: New Horizons` no longer needs the crash fix patch we were distributing in our compatibility article of the game. This was done by [improving the emulation of the Tegra X1 GPU drivers](https://github.com/yuzu-emu/yuzu/pull/4070). As a bonus, this also fixed T-posing on the later updates in `Super Smash Bros. Ultimate`!
+
+Not all games are native to the Nintendo Switch, with a portion of its library consisting of ported games from other consoles or PC. Until now, PC game ports that were originally developed for `Direct3D` and ran perfectly fine on NVN failed to run on yuzu. [Rodrigo](https://github.com/ReinUsesLisp) found a [way to translate this behavior](https://github.com/yuzu-emu/yuzu/pull/4049) to yuzu's OpenGL and Vulkan implementations, resulting in tons of western games going in game.
+
+{{< imgs
+    "./bioshock.png| Would you kindly? (BioShock Remastered)"
+  >}}
+  
+By implementing a [missing texture format](https://github.com/yuzu-emu/yuzu/pull/4005), [Rodrigo](https://github.com/ReinUsesLisp) fixed black glitches in `Xenoblade Chronicles Definitive Edition`.
```

Comment:
Just like in https://github.com/yuzu-emu/yuzu-emu.github.io/pull/176 I think "black glitches" isn't descriptive enough

Reply:
Changed.